assignments/function-conversion.js

```diff
@@ -4,22 +4,28 @@
 // console.log("Function was invoked!");
 // };
 // myFunction();
+    const myFunction = () => console.log("Function was invoked!");
+    console.log(myFunction());
 
-// let anotherFunction = function (param) {
+    // let anotherFunction = function (param) {
 //   return param;
 // };
+    const anotherFunction = (param) => {return param};
```

Comment:
if the function only has 1 parameter you don't have to use ()
you also don't need the {} or the return function for arrow functions
could be 
const anotherFunction = param => param;